packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.stories.js

```diff
@@ -0,0 +1,20 @@
+// /* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, text, number } from "@storybook/addon-knobs";
+
+import SfQuantitySelector from "./SfQuantitySelector.vue";
+
+storiesOf("Atoms|QuantitySelector", module)
+  .addDecorator(withKnobs)
+  .add("Default", () => ({
+    components: { SfQuantitySelector },
+    props: {
+      qty: {
+        default: text("qty (prop)", "0")
```

Comment:
```suggestion
        default: number("qty", 0, "Props")
```
but it shoud be a `data` :)